src/NzbDrone.Core/Indexers/BroadcastheNet/BroadcastheNetSettings.cs

```diff
@@ -37,6 +38,9 @@ public BroadcastheNetSettings()
         [FieldDefinition(3)]
         public SeedCriteriaSettings SeedCriteria { get; } = new SeedCriteriaSettings();
 
+        [FieldDefinition(4, Label = "Priority", HelpText = "Used when searching for releases, a higher priority indexer will be used if releases found have the same quality", Advanced = true)]
```

Comment:
It's not limited to searching and I don't think we want to explicitly call out the quality.

`Used when processing releases, a higher priority indexer will be used if releases found are otherwise equal`

Reply:
Suggestion based on the remark in discord: `Used when processing releases, a higher priority indexer will be used if releases found are otherwise equal. Note: only applied on indexers with the same protocol`

Reply:
A bit shorter/straight forward I think? `Used when processing releases, a higher priority indexer will be used if releases found are otherwise equal. Applies to indexers with the same protocol`

Reply:
Agreed